staging/prometheus-operator/values.yaml

```diff
@@ -1439,7 +1439,7 @@ prometheusOperator:
   ##
   configmapReloadImage:
     repository: docker.io/jimmidyson/configmap-reload
```

Comment:
Would it make sense to have this repo under our org in a fork ?

Reply:
This is a pretty popular public image, it's used by lots of places (including upstream prometheus-operator where this came from, and funnily enough even my bank use it...).

Reply:
Can we make sure to contribute this upstream? Since this is a fork, so I'm always weary of making chart changes - this can also go in kba otherwise, but since it's a cve priority maybe let's just make this change here but also open up an upstream PR. There is also a `patch` dir here that we've also added with scripts that make mesosphere-specific chart changes, to make upgrades easier, but I'm okay skipping that if this will go into upstream anyways.

tl;dr let's open up an upstream pr!

Reply:
😆 , it is all about popularity, 👍🏻 .

Reply:
@gracedo I already pushed it upstream and it's been merged - https://github.com/prometheus-operator/kube-prometheus/pull/894